### DISCUSSION TITLE
No encuentro donde hacer el challenger

### DISCUSSION THREAD

**Discussion by Pamelafer** [community/community#54728] [Programming Help]
### Body

Necesito orientación 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by VikingDDM** [comment]
Cómo puedo ayudar?

**Reply by VikingDDM** [reply]
Qué quieres exactamente

**Reply by Pamelafer** [reply]
Hacer el challenger de pyton

**Reply by VikingDDM** [reply]
Eres experto en piton?

**Reply by Pamelafer** [reply]
No
